doc/extensions/zephyr/link-roles.py

```diff
@@ -14,7 +14,7 @@
 def get_github_rev():
     tag = run_cmd_get_output('git describe --exact-match')
     if tag:
-        return tag
+        return tag.decode("utf-8")
```

Comment:
Indeed... in the 2.0.0 docs (today) ``:zephyr_file:`` links look like this: https://github.com/zephyrproject-rtos/zephyr/blob/b'zephyr-v2.0.0'/include/logging/log.h and with this patch, they're corrected to: https://github.com/zephyrproject-rtos/zephyr/blob/zephyr-v2.0.0/include/logging/log.h